ci/azure-pipelines-release.yml

```diff
@@ -0,0 +1,91 @@
+# Copyright the Hyperledger Fabric cont ributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: RELEASE-$(Date:yyyyMMdd)$(Rev:.rrr)
+trigger: none
+pr: none
+
+variables:
+  - group: credentials
+  - name: GOPATH
+    value: $(Agent.BuildDirectory)/go
+  - name: GOVER
+    value: 1.12.12
```

Comment:
This is stale now.

Reply:
https://github.com/hyperledger/fabric-ca/blob/release-1.4/ci.properties#L8

Reply:
This pipeline file is going into 1.4, when I replicate it to master I'll upgrdae